frontend/test/cypress/integration/product-list/parts_page_search.spec.ts

```diff
@@ -14,7 +14,14 @@ describe('parts page search', () => {
          .should('be.visible')
          .type('iphone');
 
+      // Wait for search result to be updated
       user.wait('@search');
+      user.wait(2000);
```

Comment:
@batbattur I think that probably adding instructions like [these](https://github.com/iFixit/react-commerce/blob/pro-users--show-pro-products/frontend/test/cypress/integration/product-list/filters.spec.ts#L16-L17) will help to avoid this arbitrary delays.

Note that `window.userLoaded` is being added in that branch, so we'll have to merge that branch first (or replicate the same changes here)

Reply:
> Note that window.userLoaded is being added in that branch, so we'll have to merge that branch first (or replicate the same changes here)

@dhmacs Oh, I see it's getting added [here](https://github.com/iFixit/react-commerce/pull/359/files#diff-f344de4d26499cec01f1344a51b8d041d21e27a457b17cae0113e03cb876caa4R41). Instead of replicating the changes here (and possibly causing merge conflicts), I would prefer adding it later that pr gets merged.

> Do we still need this? Is there some way we can determine when the results have finished updating?

@andyg0808 I tried various ways like intercepting the new product images from the search and waiting for it to finish, or checking the div elements, but it still seems a bit flaky. We still have some wait statements like [this](https://github.com/iFixit/react-commerce/blob/pro-users--show-pro-products/frontend/test/cypress/integration/product-list/filters.spec.ts#L48-L50), so maybe we can merge this for now and fix the wait statements after when the PR above gets merged?

Reply:
That works for me.